functions-app/report-psychometrician/service/psychometrician-report.service.js

```diff
@@ -15,8 +15,18 @@ const psychometricianReportService = {}
  * @param logger - function logger
  * @return {Promise<void>}
  */
-psychometricianReportService.batchProduceCacheData = async function (batchIds, logger) {
-  const checks = await psychometricianDataService.sqlFindCompletedChecksByIds(batchIds)
+psychometricianReportService.batchProduceCacheData = async function batchProduceCacheData (batchIds, logger) {
+  const checks1 = await psychometricianDataService.sqlFindCompletedChecksByIds(batchIds)
+
+  if (!checks1 || !Array.isArray(checks1) || !checks1.length) {
+    throw new Error('batchProduceCacheData(): Failed to find any checks')
+  }
+
+  // Ticket: 33198 (investigation) check for duplicates
```

Comment:
@jon-shipley sorry my comment was not submitting I was talking about this comment here.
I believe it's best not to include that reference number within the codebase.

Reply:
Updated